### DISCUSSION TITLE
Inconsistent documentation: SwitchMode 3 and 4.

### DISCUSSION THREAD

**Discussion by theorician** [arendst/Tasmota#16094] [Q&A] [answered]
### PROBLEM DESCRIPTION
The descriptions at these two URLs:
* https://tasmota.github.io/docs/Commands/#switchmode 
* https://tasmota.github.io/docs/Buttons-and-Switches/#switchmode

indicate conflicting information. That is, on one URL,
```
3 = pushbutton (default 1, 0 = toggle)
4 = inverted pushbutton (default 0, 1 = toggle) 
```
and on the other,

```
SwitchMode 3

    Set inverted push-button mode (0 = OFF (default), 1 = TOGGLE)
...

SwitchMode 4

    Set push-button mode (0 = TOGGLE, 1 = ON (default))
```

**Comment by ascillato2** [comment, marked as answer]
Nice catch !!!!

The correct one is https://tasmota.github.io/docs/Commands/#switchmode according to the code:

https://github.com/arendst/Tasmota/blob/b82e93c1937b2e9e086b57a7c37bb6aa436e4145/tasmota/include/tasmota.h#L316-L317

Fixing the docs at https://tasmota.github.io/docs/Buttons-and-Switches/#switchmode

Thanks!!!!!

**Reply by AndreKR** [reply]
@ascillato2 You fixed the titles in https://github.com/tasmota/docs/commit/b27d036f39a2359db238e3aafd3d45ad0f6a2aa6 but the texts are also swapped. In reality SwitchMode 3 toggles when the button is released (falling-edge) and SwitchMode 4 toggles when the button is pressed (rising-edge).
